Assets/Scripts/Gameplay/GameplayObjects/ServerTossedItem.cs

```diff
@@ -1,11 +1,14 @@
+using Unity.Multiplayer.Samples.BossRoom.Client;
 using Unity.Netcode;
 using UnityEngine;
-using UnityEngine.Events;
 
 namespace Unity.Multiplayer.Samples.BossRoom.Server
 {
-    public class ServerTossAttack : NetworkBehaviour
+    public class ServerTossedItem : NetworkBehaviour
     {
+        [SerializeField]
+        ClientTossedItem m_ClientTossedItem;
```

Comment:
(but yes this would be the pattern I'd see for client/server separation: like you're doing, with client code referencing server code and server code referencing client code, tightly coupled)

Reply:
(you can resolve once read, this is just a quick comment :P )